.github/workflows/release.yml

```
@@ -38,17 +38,20 @@ jobs:
         with:
           tag-template: "{VERSION}"
           version-command: |
-            cat current-version.txt
+            cat galaxy.yml|grep version:|cut -d' ' -f2
 
       - name: Install Ansible
         run: |
           pip install --constraint=.github/workflows/constraints.txt ansible
 
-      - name: Trigger a new import on Galaxy.
+      - name: Release to Ansible Galaxy.
         run: >-
```

Existing token needs to be set into env for the playbook to pass the assert
```suggestion
      - name: Release to Ansible Galaxy.
        env:
          ANSIBLE_GALAXY_TOKEN: ${{ secrets.GALAXY_TOKEN }}
        run: >-
```